pkg/reconciler/certificate/certificate_test.go

```diff
@@ -247,6 +280,10 @@ func knCert(name, namespace string) *v1alpha1.Certificate {
 	return knCertWithStatus(name, namespace, &v1alpha1.CertificateStatus{})
 }
 
+func knCertWithGeneration(name, namespace string, gen int) *v1alpha1.Certificate {
+	return knCertWithStatusAndGeneration(name, namespace, &v1alpha1.CertificateStatus{}, gen)
+}
+
 func knCertWithStatus(name, namespace string, status *v1alpha1.CertificateStatus) *v1alpha1.Certificate {
 	return &v1alpha1.Certificate{
```

Comment:
this should use the new helper, then?

Reply:
@vagababov I was thinking if I should adding another input param to `knCert(...)` and `knCertWithStatus(...)` instead of creating the new helpers `knCertWithGeneration(...)` and `knCertWithStatusAndGeneration(...)`. WDYT?

Reply:
Yeah have one all inclusive and specialize others. That keeps the actual logic in one place.